themes.json

```diff
@@ -79,7 +79,10 @@
       "toolbar-icon-selected-bg": "#ffffff",
       "toolbar-basic-icon-color": "#ffffff",
       "toolbar-basic-selected-icon-color": "#2B313B",
-      "toolbar-basic-color-hover": "#ffffff",
```

Comment:
It seems that removing this variable makes the menu button in dark mode to have the incorrect background on hover.

Reply:
Hm, it works fine here as the following pic shows. (The mouse pointer is invisible but it is on the mic button.)

![image](https://user-images.githubusercontent.com/7637832/177422317-cf05d6ee-8099-4cfb-b1a9-8d4417672572.png)

`--toolbar-basic-color-hover` refers to `--basic-color-hover` and `-basic-color-hover` is updated for the dark mode in `theme.json`.

Reply:
Humm I just checkout the branch again and did a npm ci and it seems to be alright. Not sure what happened but looks good.